Lib/inspect.py

```diff
@@ -2395,6 +2395,17 @@ class _ParameterKind(enum.IntEnum):
     def __str__(self):
         return self._name_
 
+    @property
+    def description(self):
+        param_name_mapping = {
```

Comment:
I'd move this into a global variable to make it's faster.

Also, I want to backport this patch to 3.6 and 3.7.1 so maybe we should add the property in a separate PR that will only go to 3.8.

Reply:
Okay, I will create a new PR which add a new property `description` for 3.8 after this PR is merged.